test-runner-frontend/src/Views/ReportAndCompareUtils.js

```diff
@@ -0,0 +1,206 @@
+const HIGHER_IS_BETTER = 'higherIsBetter';
```

Comment:
we can call this file utils. we don't need to be as specific.

Reply:
Done.